src/nat/eval/llm_validator.py

```
@@ -0,0 +1,330 @@
+
+        # Validate in batches to respect rate limits
+        for batch_start in range(0, len(llm_items), CONCURRENT_VALIDATION_BATCH_SIZE):
+            batch = llm_items[batch_start : batch_start + CONCURRENT_VALIDATION_BATCH_SIZE]
+
+            # Validate batch in parallel
+            validation_tasks = [_validate_single_llm(builder, llm_name, llm_config) for llm_name, llm_config in batch]
+
+            results = await asyncio.gather(*validation_tasks, return_exceptions=True)
+
+            # Process results - zip with batch to maintain llm_name association
+            for (llm_name, llm_config), result in zip(batch, results):
```

Ruff (0.14.10)

Loop control variable `llm_config` not used within loop body

Rename unused `llm_config` to `_llm_config`

(B007)